app/assets/stylesheets/components/_credentials.scss

```diff
@@ -0,0 +1,14 @@
+.credentials {
+  &-hints {
+    @include fill-parent;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets